backend/utils/psyonix_api_handler.py

```diff
@@ -137,6 +140,28 @@ def get_rank(steam_id):
     :param steam_id: steamid to get
     :return: rank, if it exists
     """
+    if len(steam_id) == 11 and steam_id[0] == 'b' and steam_id[-1] == 'b':
```

Comment:
Don't check length.

We also need to convert psyonix bots to be considered as all star.
I think we have an is bot test that you might be able to look at

Reply:
I took that from sciguy's commit where he added support for bot stats.
(https://github.com/SaltieRL/DistributedReplays/blob/6e0aed4ae0e26079e7a774f58ab56495e55f3eb1/backend/blueprints/spa_api/service_layers/player/play_style.py#L70). But I will look into a better is bot method

Reply:
All star id is not 11 it's 4